packages/fx-core/src/component/feature/tab.ts

```diff
@@ -138,10 +138,18 @@ const configTab: Action = {
   name: "fx.configTab",
   type: "function",
   plan: (context: ContextV3, inputs: InputsWithProjectPath) => {
-    return ok(["config 'teams-tab' in projectSettings"]);
+    const tabConfig = getComponent(context.projectSetting, ComponentNames.TeamsTab);
+    if (tabConfig) {
```

Comment:
I have skip adding this action if tab has been added. So I think we don't need this condition.
https://github.com/OfficeDev/TeamsFx/blob/2ddc8f81d3b25567ea2ca1febc82807613832392/packages/fx-core/src/component/feature/tab.ts#L69

Reply:
It's a double ensurence. We can just keep it.